php/class-wp-seo-settings.php

```diff
@@ -794,4 +794,4 @@ public function view_formatting_tags_help_tab() {
 function WP_SEO_Settings() {
 	return WP_SEO_Settings::instance();
 }
-add_action( 'plugins_loaded', 'WP_SEO_Settings' );
\ No newline at end of file
+add_action( 'after_setup_theme', 'WP_SEO_Settings' );
```

Comment:
If memory serves, this needs to be at priority 9, because `WP_SEO` (at priority 10) depends on it.